OSDOCS-12237-dev-console: Bug batching 4.18 dev console (#88887)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

[id="ocp-release-note-dev-console-bug-fixes_{context}"]
==== Developer Console

> * Previously, if a `PipelineRun` was using a resolver, rerun of that `PipelineRun` was having issue. With this fix, a user can rerun `PipelineRun` if it is using resolver. (link:https://issues.redhat.com/browse/OCPBUGS-45228[*OCPBUGS-45228*])

Reviewer: ```suggestion
* Previously, if a `PipelineRun` was using a resolver, rerunning that `PipelineRun` resulted in an error. With this fix, a user can rerun `PipelineRun` if it is using resolver. (link:https://issues.redhat.com/browse/OCPBUGS-45228[*OCPBUGS-45228*])
```

> * Previously, on the *Edit of Deployment* or *DeploymentConfig* in form view added duplicate `ImagePullSecrets`. With this update, editing the form does not add duplicate entries. (link:https://issues.redhat.com/browse/OCPBUGS-45227[*OCPBUGS-45227*])

Reviewer: ```suggestion
* Previously, on the if you edited a deployment config in form view of the web console, the `ImagePullSecrets` values was duplicated. With this update, editing the form does not add duplicate entries. (link:https://issues.redhat.com/browse/OCPBUGS-45227[*OCPBUGS-45227*])
```

Author: :heavy_check_mark:

* Previously, when you searched on the `OperatorHub` or another catalog, you would experience periods of latency between each key press. With this update, the input on the catalog search bars are debounced. (link:https://issues.redhat.com/browse/OCPBUGS-43799[*OCPBUGS-43799*])

> * The *Getting started* section with resources was converted to a collapse and expandable section from {product-title} 4.16, but there was no option to close the *Getting started* section. With this change, user can close the *Getting started* section. (link:https://issues.redhat.com/browse/OCPBUGS-38860[*OCPBUGS-38860*])

Reviewer: Also, the Getting started resources tile is in the Administrator perspective, but listed here in the Developer Console section. Is that correct?
```suggestion
* Previously, there was no option to close the *Getting started resources* section. With this change, user can close the *Getting started resources* section. (link:https://issues.redhat.com/browse/OCPBUGS-38860[*OCPBUGS-38860*])
```

> * Previously, when cronjobs were created, the creation of pods happens very quickly, causing the component that is constantly fetching new pods off the cronjob to fail. With this update, a 3 second delay was added before starting to fetch the pods of the cronjob. (link:https://issues.redhat.com/browse/OCPBUGS-37584[*OCPBUGS-37584*])

Reviewer: ```suggestion
* Previously, when cronjobs were created, the creation of pods happens too quickly, causing the component that is fetches new pods off the cronjob to fail. With this update, a 3 second delay was added before starting to fetch the pods of the cronjob. (link:https://issues.redhat.com/browse/OCPBUGS-37584[*OCPBUGS-37584*])
```

> * Previously, resources created when a new user is created was not removed automatically when the user was deleted. This caused clutter on the cluster with configuration maps, roles, and role-bindings. With this update, `ownerRefs` was added to the resources, so they are cleared once the user is deleted and the cluster no longer clutters with users. (link:https://issues.redhat.com/browse/OCPBUGS-37560[*OCPBUGS-37560*])

Reviewer: ```suggestion
* Previously, resources created when a new user is created were not removed automatically when the user was deleted. This caused clutter on the cluster with configuration maps, roles, and role-bindings. With this update, `ownerRefs` was added to the resources, so they are cleared once the user is deleted. (link:https://issues.redhat.com/browse/OCPBUGS-37560[*OCPBUGS-37560*])
```
